xml/x509_subjects.xml

```diff
@@ -878,7 +878,7 @@
     <param pos="0" name="os.device" value="WAP"/>
   </fingerprint>
   <fingerprint pattern="^CN=axis-([a-fA-F0-9]{12}),O=Axis Communications AB$">
-    <description>OpenWRT WAP</description>
+    <description>OpenWRT WAP - Axis Communications variant</description>
```

Comment:
I suspect this was a copy-past problem from the fingerprint above.  I don't believe this is OpenWRT.

Reply:
Thanks, I should have caught that. I caught similar in one or two other places.